cc_plugin_glider/glider_dac.py

```diff
@@ -640,6 +731,7 @@ def check_standard_names(self, dataset):
         out_of = 1
         score = 0
         messages = []
+        # only check salinity?
```

Comment:
i've got the same question. Doesn't your implementation using expected_format.py check for correct standard names?